### PR TITLE
chore: stacks UI utils removal

### LINKED ISSUE #4164: Chore: Remove emotion + stacks UI
This task is to track work on refactoring out:
- emotion now we have panda
- stacks UI

### PR DESCRIPTION
**Pull request by pete-watters**
> Try out this version of the Leather — download [extension builds](https://github.com/leather-wallet/extension/actions/runs/6147952338).

This PR covers migrating some code from `@stacks/ui-utils`

### CODE REVIEW

File: src/app/common/utils/stacks-ui/tx.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+var txTypeNamesMap =
+  ((_txTypeNamesMap = {}),
+  (_txTypeNamesMap[TransactionType.SMART_CONTRACT] = 'Contract deploy'),
+  (_txTypeNamesMap[TransactionType.CONTRACT_CALL] = 'Function call'),
+  (_txTypeNamesMap[TransactionType.TOKEN_TRANSFER] = 'Token transfer'),
+  (_txTypeNamesMap[TransactionType.COINBASE] = 'Coinbase'),
+  (_txTypeNamesMap[TransactionType.POISON_MICROBLOCK] = 'Poison-microblock'),
+  _txTypeNamesMap);
+function getTxTypeName(tx) {
+  return txTypeNamesMap[tx.tx_type];
```
Comment (kyranjamie) [inline src/app/common/utils/stacks-ui/tx.ts:120]: Why copy the resulting js vs the ts source?

Comment (pete-watters) [reply src/app/common/utils/stacks-ui/tx.ts:120]: I didn't know there was TS source, maybe I'm looking in the wrong place. 

I was navigating through the wallet `node_modules/@stacks/ui-utils/dist` 

I can dig through the actual repo and find the TS, thanks!

Comment (kyranjamie) [reply src/app/common/utils/stacks-ui/tx.ts:120]: https://github.com/hirosystems/ui

**Comment by pete-watters** [comment]
Closing off for now, can revisit

**Pull request closed without merging**